### PR TITLE
Add click(selector) example

### PR DESCRIPTION
**Pull request by dominicfraser**
Add interaction example to make it clear to those not familiar with Puppeteer's API that these are supported, including an additional example of how the `.result()` call can be used.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -461,5 +461,23 @@ In this example, you can specify the custom path for storing images.
...
+    .init()
+    .newPage()
+    .goto('https://nimasoroush.github.io/differencify/')
+    .waitFor(2000)
+    .click('[id=js-repo-pjax-container] div nav span a')
+    .waitFor(2000)
+    .url()
+    .result((url) => {
+      expect(url).toContain('pulls');
+    })
```
Comment (NimaSoroush) [inline API.md]: worth adding 

```
.screenshot()
.toMatchSnapshot()
```
as differencify is really about snapshot testing

**Comment by NimaSoroush** [comment]
BTW, Readme mentions [Differencify matches Puppeteer's API completely. Look at API.md for more details](https://github.com/NimaSoroush/differencify#usage). Click is one the many API calls supported by Puppeteer. But still worth keeping the example

File: API.md (hunk 1 above)
Comment (dominicfraser) [reply API.md]: In this case the screenshot would change each time, as the url it navigates to is for a dynamic page (different PRs may be open each time it is ran). So the example is really showing that if no other solution is being used it also has the capabilities to test other things as well as visual tests.

Comment (NimaSoroush) [reply API.md]: Can we click on `wiki` tab instead which is more or less static

Comment (dominicfraser) [reply API.md]: 👍

Comment (NimaSoroush) [reply API.md]: Awesome. Thanks for contribution

**Pull request merged**